apps/src/lib/client/utils.rs

```diff
@@ -345,10 +333,31 @@ pub async fn join_network(
         .await
         .unwrap();
     }
+    if !dont_prefetch_wasm {
```

Comment:
Double negatives. Tsk tsk

Reply:
Yeah we could have a style guide around this.